src/Fabulous.XamarinForms/Views/Controls/Button.fs

```diff
@@ -154,8 +154,8 @@ type ButtonModifiers =
         res
 
     [<Extension>]
-    static member inline imageSource(this: WidgetBuilder<'msg, #IButton>, light: string, ?dark: string) =
-        this.AddScalar(Button.ImageSource.WithValue(AppThemeValues<string>.create (light, dark)))
+    static member inline imageSource(this: WidgetBuilder<'msg, #IButton>, light: ImageSource, ?dark: ImageSource) =
```

Comment:
Could you add the other extension as well for path, uri and stream, please?

Reply:
ImageSource will will for all of them , You can even mix them . So I do not think is necessary to add them : 

``` fsharp 
Button(text, onClicked = command)
     .imageSource(
          ImageSource.FromUri(Uri("")),
          ImageSource.FromFile(Images.OnboardingTick.Dark)
    )
```

Reply:
I think we should add them mainly for consistency between all controls.
If I can write `Image("dark.png")`, I should be able to do `Button(...).imageSource("dark.png")` as well.

Maybe we need to rename `ImageSource` to `Image` in that case.

Reply:
I have added support for path uri and stream now